### PR TITLE
TECS initialization and misc cleanup

### PR DESCRIPTION
**Pull request by dagar**
- TECS initialization
  - subtract pitchsp_offset from _pitch used to initialize TECS
  - constrain pitch to min, max (eg bad throws)
  - TAS instead of EAS where appropriate

- remove unused COM_DL_LOSS_EN in SITL vtol configs
- remove unused vehicle_status from vtol_att_control
- fix land_detector status string

### CODE REVIEW

**Comment by sanderux** [comment]
@dagar have you test flown this?

File: src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp [hunk 1]
```diff
@@ -2476,7 +2473,7 @@ void FixedwingPositionControl::tecs_update_pitch_throttle(float alt_sp, float v_
 					      || mode == tecs_status_s::TECS_MODE_LAND_THROTTLELIM));
 
 	/* Using tecs library */
-	float pitch_for_tecs = _pitch;
+	float pitch_for_tecs = _pitch - _parameters.pitchsp_offset_rad;
```
Comment (dagar) [inline src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp]: @sanderux FYI - this is the only important change

**Comment by dagar** [comment]
Only SITL so far. The only important part of this PR is the pitch for tecs. If you take a look at transition to FW you'll see the initial pitch setpoint from TECS offset.
The rest is trivial/cosmetic readability stuff.

**Comment by AndreasAntener** [comment]
@dagar can you explain the TAS/EAS changes?

**Comment by dagar** [comment]
_integ5_state is TAS, but it was initialized with EAS. _TAS_dem is 0
initially.
I'm aware that fw_pos_ctrl_l1 still uses 1 for EAS2TAS, but I'm planning
ahead.

**Pull request merged**